pkg/storaged/content-views.jsx

```diff
@@ -372,99 +372,52 @@ function append_partitions(client, rows, level, block) {
     var device_level = level;
 
     var is_dos_partitioned = (block_ptable.Type == 'dos');
-    var partitions = client.blocks_partitions[block.path];
 
     function append_free_space(level, start, size) {
-        // There is a lot of rounding and aligning going on in
-        // the storage stack.  All of udisks2, libblockdev,
-        // and libparted seem to contribute their own ideas of
-        // where a partition really should start.
-        //
-        // The start of partitions are aggressively rounded
-        // up, sometimes twice, but the end is not aligned in
-        // the same way.  This means that a few megabytes of
-        // free space will show up between partitions.
-        //
-        // We hide these small free spaces because they are
-        // unexpected and can't be used for anything anyway.
-        //
-        // "Small" is anything less than 3 MiB, which seems to
-        // work okay.  (The worst case is probably creating
-        // the first logical partition inside a extended
-        // partition with udisks+libblockdev.  It leads to a 2
-        // MiB gap.)
-
         function create_partition() {
             FormatDialog.format_dialog(client, block.path, start, size, is_dos_partitioned && level <= device_level);
         }
 
-        if (size >= 3*1024*1024) {
-            var btn = (
-                <StorageButton onClick={create_partition}>
-                    {_("Create Partition")}
-                </StorageButton>
-            );
-
-            var cols = [
-                <span className={"content-level-" + level}>{utils.fmt_size(size) + " " + _("Free Space")}</span>,
-                "",
-                { element: btn, tight: true }
-            ];
-
-            rows.push(
-                <CockpitListing.ListingRow columns={cols}/>
-            );
-        }
+        var btn = (
+            <StorageButton onClick={create_partition}>
+                {_("Create Partition")}
+            </StorageButton>
+        );
+
+        var cols = [
+            <span className={"content-level-" + level}>{utils.fmt_size(size) + " " + _("Free Space")}</span>,
```

Comment:
It might be good to use cockpit's formatting function here to combine "free space" with the size. Depending on the language, it might be ordered differently.